### PR TITLE
Add support for multiple timers and other features

### PR DESCRIPTION
**Pull request by millerdev**
Features added
- Time of day parser (ex: 5:30pm)
- Multiple timers
- Single notification per timer
- Persistent timers (alert until closed or stopped)
- Ability to stop running timers
- Basic documentation

Persistent timers are currently optional and not the default. I think it would make sense to make them the default and have the old single alert timer be optional (or even remove the option completely).

Tests were added for all new features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,48 @@
...
+
+pip install websocket-client python-Levenshtein
+pip install pytest pytest-pep8 freezegun
+```
+
+Run tests
+
+```sh
+pytest
+```
```
Comment (gornostal) [inline README.md:49]: Thanks for the readme updates too :+1:

File: tests/timer/test_TimerExtension.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "<start at 12:00 PM / Select to stop>",
+        "<Time is up! at 12:05 PM / Select to stop>",
+    ]
+    stop_query[0].enter()
+    assert notifications == [
+        "<start at 12:00 PM>",
+        "<Time is up! at 12:05 PM>",
+        "<Timer stopped / start at 12:00 PM>",
+    ]
+    assert launcher.query("") == ["<Time is up! at 12:05 PM / Select to stop>"]
```
Comment (gornostal) [inline tests/timer/test_TimerExtension.py:58]: You have really put a lot of effort into tests. I appreciate that!

**Review by gornostal — APPROVED** [review]
Wow! These improvements look awesome!
Merging.

**Comment by millerdev** [comment]
@gornostal thanks!

FYI, I found a bug just now where "persistent" timers sometimes do not get stopped, and Ulauncher must be restarted to stop the alerts. I will work on a fix ASAP (hopefully later today).

**Pull request merged**